src/utils/getPeoplePrepeared.tsx

```diff
@@ -0,0 +1,71 @@
+/* eslint-disable operator-linebreak */
+import { Person } from '../types';
+
+export enum SortField {
+  NAME = 'name',
+  SEX = 'sex',
+  BORN = 'born',
+  DIED = 'died',
+}
+
+export function getPeoplePrepeared(
+  people: Person[],
+  query: string,
+  selectedSex: string,
+  centuries: string[],
+  sortField: SortField | null,
+  sortOrder: string,
+): Person[] {
+  let prepearedPeople = [...people];
+
+  if (query) {
+    const prepearedQuery = query.toLowerCase().trim();
+
+    prepearedPeople = prepearedPeople.filter(person => {
+      return (
+        person.name.toLowerCase().includes(prepearedQuery) ||
```

Comment:
Same here don't forget to use [destruction assigment](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment) will improve code readability